convert2rhel/pkgmanager/handlers/yum/__init__.py

```diff
@@ -67,7 +67,7 @@ def _resolve_yum_problematic_dependencies(output):
             packages_to_remove.append(str(resolve_error[0]).replace(" ", ""))
 
     if packages_to_remove:
-        packages_to_remove = set(packages_to_remove)
+        packages_to_remove = list(set(packages_to_remove))
```

Comment:
I would add a comment saying that we need to make each item in the list unique. But not a blocker, looks good
```suggestion
        # Need to make each item in the list unique, one pkg can be present more times in the list.
        packages_to_remove = list(set(packages_to_remove))
```

Reply:
Lovely, thanks!